src/layers/MultiscaleImageLayer/MultiscaleImageLayer.js

```diff
@@ -218,19 +200,7 @@ const MultiscaleImageLayer = class extends CompositeLayer {
         getTileData: [loader, loaderSelection]
       },
       onTileError: onTileError || loader[0].onTileError,
-      opacity,
-      colormap,
-      viewportId,
-      onHover,
-      pickable,
-      unprojectLensBounds,
-      isLensOn,
-      lensSelection,
-      lensBorderColor,
-      lensBorderRadius,
-      modelMatrix,
-      transparentColor,
-      onViewportLoad
```

Comment:
Explicitly declaring these props was problematic in the tests. Now that the docs are forced to be correct, I think it's OK to get rid of this which was mostly a reminder to myself anyway.